src/rust/kafka-executor/src/event_consumer.rs

```diff
@@ -0,0 +1,82 @@
+use rdkafka::consumer::{StreamConsumer, DefaultConsumerContext, ConsumerContext, Consumer};
+use rdkafka::util::{DefaultRuntime, AsyncRuntime};
+use grapl_config::env_helpers::FromEnv;
+use rdkafka::config::{RDKafkaLogLevel, FromClientConfig};
+use std::sync::Arc;
+
+const KAFKA_CONSUMER_TOPIC: &str = "KAFKA_CONSUMER_TOPIC";
+const KAFKA_CONSUMER_BROKERS: &str = "KAFKA_CONSUMER_BROKERS";
+const KAFKA_CONSUMER_GROUP_ID: &str = "KAFKA_CONSUMER_GROUP_ID";
+const KAFKA_CONSUMER_CLIENT_ID: &str = "KAFKA_CONSUMER_CLIENT_ID";
+const KAFKA_CONSUMER_SESSION_TIMEOUT_MS: &str = "KAFKA_CONSUMER_SESSION_TIMEOUT_MS";
+const KAFKA_CONSUMER_ENABLE_AUTO_COMMIT: &str = "KAFKA_CONSUMER_ENABLE_AUTO_COMMIT";
+const KAFKA_CONSUMER_MAX_POLL_INTERVAL_MS: &str = "KAFKA_CONSUMER_MAX_POLL_INTERVAL_MS";
+const KAFKA_CONSUMER_REQUEST_TIMEOUT_MS: &str = "KAFKA_CONSUMER_REQUEST_TIMEOUT_MS";
+
+pub struct EventConsumer<C=DefaultConsumerContext, R=DefaultRuntime>
```

Comment:
Would "KafkaConsumer" work here? Similarly, this file can be renamed to "kafka_consumer.rs". The reason I suggest this is that this isn't so much about events (we don't really have a coherent "Event" type here).